packages/init/__tests__/configure-project.test.ts

```diff
@@ -87,6 +93,9 @@ describe("configure-project", () => {
   const mkdirMock = mkdir as jest.Mock;
 
   test("it should create a new manifest if it doesn't exist yet", () => {
+    // vol.fromJSON({
```

Comment:
:)

Reply:
yeah I was going to refactor this so we get more visibility on the core by not mocking it